.github/dependabot.yml

```diff
@@ -11,7 +11,11 @@ updates:
     schedule:
       interval: "daily"
   - package-ecosystem: "gomod"
-    directory: "/build"
```

Comment:
why? what if there is a new release of `goyek`?

Reply:
Fixed with https://github.com/signalfx/splunk-otel-go/pull/117